lib/api/v1/user-controller.js

```diff
@@ -34,15 +35,19 @@ module.exports = function(app, user) {
   app.use(express.json());
 
   // Get a list of all Users
-  app.get('/api/v1/user', user.can('manage users'), function(req, res) {
-    User.find({}, '-password', function(err, users) {
+  app.get('/api/v1/user', user.can('view users'), function(req, res) {
+    var query = {};
+    if (req.user && (new UserPermissions(req.user)).can('not view other users'))
```

Comment:
I agree with the negative not being a great idea, I will make it positive. 

In my understading for privacy and authorization viewers shouldn't be able to list all users. However, in the current implementation the incidents, and others, angular controller does require User.query() to grab all the users for things like filtering. See [routes.js](https://github.com/TID-Lab/aggie/blob/651b37318160e656d9dc307b5b27fa819aa6dce9/public/angular/js/routes.js#L130). That's the reason why everybody has access to this endpoint. 

I couldn't think of a better way to get around that requirement.